src/components/organisms/UiList/_internal/UiListItemAffix.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <component
+    :is="listItemAffixComponent"
+    :class="[
+      'ui-list-item-affix',
+      { 'ui-button--text': isButton },
+    ]"
+  >
+    <slot
+      v-bind="{
+        label,
+        icon,
+        isButton,
+        hasIcon,
+        iconAttrs,
+        labelAttrs,
+      }"
+    />
+  </component>
+</template>
+
+<script setup lang="ts">
+import {
+  computed,
+  defineAsyncComponent,
+  useAttrs,
+} from 'vue';
+import UiIcon, { type IconAttrsProps } from '../../../atoms/UiIcon/UiIcon.vue';
+
+import type {
+  DefineAttrsProps,
+  HTMLTag,
+  Icon,
+} from '../../../../types';
+
+export interface ListItemAffixProps {
+  /**
+   * Use this props to set list item suffix tag.
+   */
+  tag?: HTMLTag;
+  /**
+   * Use this props to set label.
+   */
+  label?: string;
+  /**
+   * Use this props to set icon.
+   */
+  icon?: Icon;
+  /**
+   * Use this props to pass attrs for suffix UiIcon.
+   */
+  iconAttrs?: IconAttrsProps;
+  /**
+   * Use this props to pass attrs for label element.
+   */
+  labelAttrs?: DefineAttrsProps<null>;
+}
+export type ListItemAffixAttrsProps = DefineAttrsProps<ListItemAffixProps>
+const props = withDefaults(defineProps<ListItemAffixProps>(), {
+  tag: undefined,
+  label: '',
```

Comment:
Shouldn't the default value be `undefined` if the user has nothing passed?

Reply:
for `label`? label is string then default value should be `''`